asserts/ifacedecls.go

```diff
@@ -293,6 +295,9 @@ func compileAltAttrMatcher(cc compileContext, l []interface{}) (attrMatcher, err
 }
 
 func (matcher altAttrMatcher) feature(flabel string) bool {
+	if flabel == altAttrMatcherFeature {
+		return true
+	}
```

Comment:
the thinking is that all rules now will need format 5 to support this because the previous behavior of the alternative matcher is wrong for these sorts of interfaces?

If that's not the case, I'm slightly confused how this `feature(...)` always returning true for the alternative attribute is used in conjunction with the `snapDeclarationFormatAnalyze()` below as it seems like this will ensure that every snap-declaration now requires format 5

Reply:
the feature check will return true only if we reach here but we reach here only if the compiled tree contains a node of this kind (altAttrMatcher), so it's not true that all the new snap-declarations will have format 5, only the ones which such construct

Reply:
ah okay that makes sense, it was difficult to follow if it would get to that point or not